src/components/panel/panel.js

```diff
@@ -1153,20 +1156,24 @@ MdPanelRef.prototype.open = function() {
 
 /**
  * Closes the panel.
+ * @param {string} closeEvent The event type that triggered the close.
```

Comment:
This parameter would most likely not be a string. Events are normally `MouseEvent`, or `AngularEvent`.